e2e-tests/README.md

```
@@ -140,7 +140,11 @@ $ podman push quay.io/konflux_ui_qe/konflux-ui-tests:mytag
 Test automation allows us to test early in the process and discover bugs before pushing code to the staging environment.
 
 ### PR checks
-Cypress E2E tests run on pull requests using a [GitHub Action](https://github.com/konflux-ci/konflux-ui/blob/main/.github/workflows/pr-check.yaml). To be able to run the PR check job, you have to be a public member of `konflux-ci` organization.
+Cypress E2E tests run on pull requests using a [GitHub Action](https://github.com/konflux-ci/konflux-ui/blob/main/.github/workflows/pr-check.yaml). PR checks are allowed when at least one of the following conditions is met:
+
+- PR author is `PR author `
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)